ibm/service/cdtektonpipeline/resource_ibm_cd_tekton_pipeline_trigger_property.go

```diff
@@ -278,8 +279,8 @@ func resourceIBMCdTektonPipelineTriggerPropertyUpdate(context context.Context, d
 			" The resource must be re-created to update this property.", "name"))
 	}
 	if d.HasChange("type") {
-		replaceTektonPipelineTriggerPropertyOptions.SetType(d.Get("type").(string))
```

Comment:
Once we mark the attribute as ForceNew true this block of code `if d.HasChange("type") {}` is not required .

Reply:
This is generated by the [openapi-sdk-generator](https://github.ibm.com/CloudEngineering/openapi-sdkgen), so may be a bug in the generator causing it to add this block erroneously

Reply:
Ok . We will talk to generator team

Reply:
Will this prevent this PR from being merged unless the block is removed?